internal/model/tabs/jobfromtemplate/jobfromtemplate.go

```diff
@@ -60,26 +62,39 @@ func GetTemplate(name string, l *log.Logger) tea.Cmd {
 
 type TemplatesListRows []table.Row
 
-var (
-	DefaultTemplatePaths = []string{
-		"/etc/slurmcommander-dev/templates",
-	}
-)
-
 func GetTemplateList(paths []string, l *log.Logger) tea.Cmd {
 
 	return func() tea.Msg {
 		var tlr TemplatesListRows
 		for _, p := range paths {
+			l.Printf("GetTemplateList reading dir: %s\n", p)
 			files, err := os.ReadDir(p)
 			if err != nil {
 				l.Printf("GetTemplateList ERROR: %s\n", err)
 			}
 			for _, f := range files {
 				l.Printf("GetTemplateList INFO files: %s %s\n", p, f.Name())
-				// TODO: check suffix, if .sbatch, append to tlr
-				// if suffix=".descr" then read content and use as description
-				tlr = append(tlr, table.Row{f.Name(), "Description", p + "/" + f.Name()})
+				// if suffix=".desc" then read content and use as description
```

Comment:
one alternative instead of having a seperate .desc file for the description, is to include it as a #comment in the .sbatch file itself (maybe with a marker #DESC_START...#DESC_END)

Reply:
Did cross my mind, was tempting also to expand this basic idea with a wider one, a small set of tags for a template creators, something like:
`--job-name={string:"Enter job name":"defaultname"}` and then parse those, and instead of editing the template, user just gets a form/questionnaire with these fields to fill out... Perhaps in some future version.
This i went in the end simply because it's simplest, and quickest, and i need such solutions now :smile: